### PR TITLE
Initialize of project

### PR DESCRIPTION
**Pull request by Sherinpour**

### CODE REVIEW

**Comment by mrp97** [comment]
@Sherinpour Please change the name of this pull request to a meaningful title.

File: index.js [hunk 1]
```diff
@@ -13,47 +13,93 @@
...
 })();
 
-//Add todo item
-function addItem(){
-    let now = new Date();
-    let todoItem = todoInput.value;
+
+/////////////////////Load todo items
+window.onload = function() {
+     let PreviousTodos = Object.entries(localStorage);
```
Comment (mrp97) [inline index.js]: If another part of code wants to use the localStorage, our code may be broken. So we should use localStorage in a safe way.

Do you have any idea about how to do it?

File: index.js [hunk 2]
```diff
@@ -13,47 +13,93 @@
...
+    if(PreviousTodos.length !== 0){
+        for(let i=0; i<PreviousTodos.length; i++){
+            let id = PreviousTodos[i][0];
+            let inf = JSON.parse(localStorage.getItem(id));
```
Comment (mrp97) [inline index.js]: What's the meaning of `inf` variable? 

It's better to use meaningful name for our variables in the code. Please read this link:
https://github.com/ryanmcdermott/clean-code-javascript#variables (Specifically the Variables section)

And then try to revise the code and enhance the variable names.

For example, the variable `id` could be renamed to `previousTodos`

File: index.js [hunk 3]
```diff
@@ -13,47 +13,93 @@
...
     checkboxes.forEach((checkbox) => {
         checkbox.addEventListener('click', () => {
             if(checkbox.checked){
                 checkbox.parentElement.style.textDecoration = "line-through";
+                let array = [checkbox.nextSibling.nodeValue, checkbox.parentElement.parentElement.firstChild.children[0].checked]
+                localStorage.setItem(checkbox.parentElement.parentElement.id, JSON.stringify(array));
             }else{
                 checkbox.parentElement.style.textDecoration = "none";
+                let array = [checkbox.nextSibling.nodeValue, checkbox.parentElement.parentElement.firstChild.children[0].checked]
+                localStorage.setItem(checkbox.parentElement.parentElement.id,JSON.stringify(array));
```
Comment (mrp97) [inline index.js]: Here in both the if and else block, we have a copied code. We can enhance it in way to not copy the two lines of code. Do you have any idea about that?

Hint: Try to define a CSS class `todo-item.checked` for example and apply it to the `tr` element.

File: index.html [hunk 4]
```diff
@@ -22,14 +23,15 @@
...
+                <tbody class="todoListBody">
                     
                 </tbody>
             </table>
+
             
                 <div class="input-group mb-3" id="inputBox">
                     <input type="text" id="todoInput" class="form-control" placeholder="Type something..." aria-describedby="add">
-                    <span class="input-group-text" id="add">
+                    <span class="input-group-text" id="add" onclick="addItem(Math.floor(Date.now() / 1000), document.getElementById('todoInput').value, false, new Date() )">
```
Comment (mrp97) [inline index.html]: @Sherinpour 
What is the `Math.floor(Date.....)`??
When I read this line of code, I should think about it to recognize the logic behind that.

One of the metric for quality of the code is WTF / m. (What the Fuck! per minute)

You can read about it in this link: http://reviewthecode.blogspot.com/2016/01/wtf-per-minute-actual-measurement-for.html

After reading this link, try to make this line, more cleaner.

**Comment by mrp97** [comment]
I think we can merge this pull request. From now on, try to  keep `main` branch as a stable version, and do the development in `develop` branch. And also use the `main` branch for the Github Pages.

**Comment by mrp97** [comment]
> I think we can merge this pull request. From now on, try to keep `main` branch as a stable version, and do the development in `develop` branch. And also use the `main` branch for the Github Pages.

Any Updates @Sherinpour ?

**Comment by Sherinpour** [comment]
> 

yes. I did everything and updated.

**Pull request merged**